src/plots/cartesian/axes.js

```diff
@@ -2861,7 +2867,9 @@ axes.drawGrid = function(gd, ax, opts) {
     grid.attr('transform', opts.transFn)
         .attr('d', opts.path)
         .call(Color.stroke, ax.gridcolor || '#ddd')
-        .style('stroke-width', ax._gw + 'px');
+        .style('stroke-width', ax._gw + 'px')
+        .style({ opacity: 100 }); // ensure visible
```

Comment:
```suggestion
        .style('opacity', 1); // ensure visible
```
- use the 2-argument form, as that's the only form available in later d3 versions (same comment elsewhere in this PR)
- opaque is 1, not 100, right?

Reply:
Addressed in https://github.com/plotly/plotly.js/pull/5550/commits/60fe3d2a39bffffc70596b757b42048d635b02d8.